src/content/docs/nrql/using-nrql/subqueries-in-nrql.mdx

```diff
@@ -59,9 +53,9 @@ Here are some example subqueries:
     id="numeric-conditions"
     title="Numeric conditions"
   >
-    Any query which returns a single numeric value, such as those using the aggregators `average()`, `count()`, `uniqueCount()`, or `latest()`, can be used in numeric conditions.
+    You can use any query which returns a single numeric value, such as those using the aggregators `average()`, `count()`, `uniqueCount()`, or `latest()` in numeric conditions.
 
-    Find the count of transactions whose duration was greater than average:
+    The query below finds the count of transactions whose duration was greater than average:
```

Comment:
I don't think you need this "The query below finds" phrasing. I slightly prefer the previous wording